config/routes.rb

```diff
@@ -3,6 +3,7 @@
 
   get '/', to: 'home#index'
   post '/download', to: 'home#download'
+  get '/download', to: 'home#download'
```

Comment:
Do you remember why we have both a `post` and a `get` request for the same route?

Reply:
Oops! Sorry about that! It is my mistake!